pkg/querier/batch/chunk.go

```diff
@@ -32,8 +32,6 @@ func (i *chunkIterator) reset(chunk GenericChunk) {
 	i.batch.Index = 0
 }
 
-// Seek advances the iterator forward to the value at or after
-// the given timestamp.
```

Comment:
What's the reason for entirely dropping the doc comment of this method?

Reply:
None of the other methods on `chunkIterator` have doc comments, and this comment was potentially confusing considering the slightly different comment on the `iterator` interface. So I opted to remove this comment and make the comment on `iterator` clearer.